scrapers/vixenNetwork.py

```diff
@@ -14,7 +14,7 @@ def log(*s):
 
 
 def get_from_url(url_to_parse):
-    m = re.match(r'(?:https?://)?(?:www\.)?((\w+)\.com)/([a-z0-9-]+)', url_to_parse)
+    m = re.match(r'(?:https?://)?(?:www\.)?((\w+)\.com)/videos/([a-z0-9-]+)', url_to_parse)
```

Comment:
while that would work for scenes that have the new urls it wont work for old ones
the below should work with old and updated urls
```suggestion
    m = re.match(r'(?:https?://)?(?:www\.)?((\w+)\.com)/(?:videos/)?([a-z0-9-]+)', url_to_parse)
```